contracts/DomainStorage.sol

```diff
@@ -0,0 +1,114 @@
+// SPDX-License-Identifier: UNLICENSED
```

Comment:
for all contract 
all function is public, it is better to set it to external
same thing but cost less gas